src/components/Header/Header.js

```diff
@@ -40,23 +101,21 @@ export default function Header({
                 </HStack>
                 <HStack>
                   <FormControl>
-                    <Input
-                      placeholder="Workspace Name"
-                      value={workspace ?? ""}
-                      onChange={(e) => {
-                        saveWorkspace(e.target.value);
-                      }}
+                    <Select
+                      options={ allOrganizations.map( ( organization ) => ( { label: organization.name, value: organization.id } ) ) }
+                      onChange={ ( organization ) => setChosenOrganization(organization.value) }
+                    />
+                  </FormControl>
+                  <FormControl>
+                    <Select
+                      options={ allOrganizations.find( ( organization ) => chosenOrganization === organization.id )?.workspaces.map( ( workspace ) => ( { label: workspace.name, value: workspace.id } ) ) }
```

Comment:
Would be nice to add Prettier at some point :)

Reply:
Updated, thank you!